src/pages/developing/frameworks/react-wrapper.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Frameworks
+description: Carbon for IBM.com Web Components, using Custom Elements v1 and Shadow DOM v1 specs.
+tabs: ['Web Components', 'React', 'React Wrapper']
+---
+
+import { Link } from 'gatsby';
+
+<PageDescription>
+
+Our team builds new components for Carbon for IBM.com as web components first. Anything that currently does not exist in the React framework, are provided as web components with a React wrapper for seamless integration into a React application.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Install via import</AnchorLink>
+  <AnchorLink>Getting started</AnchorLink>
+  <AnchorLink>NextJS template</AnchorLink>
+  <AnchorLink>Available components</AnchorLink>
+  <AnchorLink>Using with Carbon Web Components</AnchorLink>
+</AnchorLinks>
+
+## Install via import
+
+Using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon/ibmdotcom-web-components lit-html lit-element
+```
+
+Using [Yarn](https://yarnpkg.com/en/):
+
+```bash
+yarn add @carbon/ibmdotcom-web-components lit-html lit-element
+```
+
+### lit-html / lit-element
+
+`@carbon/ibmdotcom-web-components` uses [`lit-html`](https://lit-html.polymer-project.org) for reactive templating on
+top of the Web Components standard, and [`lit-element`](https://lit-element.polymer-project.org) for reactive
+properties/attributes on top of `lit-html`. These are `peerDependencies` of the project, so they would need to be
+installed separately.
+
+## Getting started
+
+1. These components require the use of [Webpack](https://webpack.js.org/) in
+   your project. See our
+   [`webpack.config.js`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/examples/codesandbox/components-react/button-group/webpack.config.js)
+   for a simple example configuration.
+2. Since these componente are being generated from the Carbon for IBM.com Web Component library the components will import all of the styles themselves.
```

Comment:
```suggestion
2. Since these components are being generated from the Carbon for IBM.com Web Component library, the components will import all of the styles themselves.
```